src/components/RoutePDF/RoutePDF.jsx

```diff
@@ -0,0 +1,84 @@
+import React, { useEffect, useState } from 'react';
+import { PropTypes } from 'prop-types';
+import { Page, Text, View, Document, StyleSheet, Svg, Line } from '@react-pdf/renderer';
```

Comment:
🚫 **[eslint]** <[import/no-unresolved](https://github.com/import-js/eslint-plugin-import/blob/v2.27.5/docs/rules/no-unresolved.md)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Unable to resolve path to module '@react-pdf/renderer'.